### DISCUSSION TITLE
[5.0] EntryTypes: Custom Color

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#14342] [General]
It would be very useful to be able to create own color schemes for the EntryTypes, with the colors that would match the companies colors for which the website was created. 

And also, for example, to create a color scheme that is monotone or in the same color range.

**Comment by brandonkelly** [comment]
Considered this, but it’s really more of a “hue” selection than “color”: whichever color you select, we have multiple specific shades of it that we’re going to use depending on the context (card BG, selected card BG, Matrix block border, icon color, and text color). So if it were to be customizable, we’d need to add settings for each of those shades, which would be a lot.

(Simply taking a single color and adjusting its brightness doesn’t look very good, so that’s not an option.)
